internal/core/CMakeLists.txt

```diff
@@ -16,15 +16,23 @@
 
 cmake_minimum_required( VERSION 3.18 )
 
+if ( APPLE )
+    set( CMAKE_CROSSCOMPILING TRUE )
+    set( RUN_HAVE_GNU_POSIX_REGEX 0 )
+    set( CMAKE_C_COMPILER "/usr/local/opt/llvm/bin/clang" )
+    set( CMAKE_CXX_COMPILER "/usr/local/opt/llvm/bin/clang++" )
+endif ()
+
 add_definitions(-DELPP_THREAD_SAFE)
 set(CMAKE_POSITION_INDEPENDENT_CODE ON)
 message( STATUS "Building using CMake version: ${CMAKE_VERSION}" )
 
-project(core LANGUAGES C CXX)
+project(core)
 include(CheckCXXCompilerFlag)
-if (${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
```

Comment:
if use APPLE,  don't use MACOS at other places, keep style is uniform

Reply:
This line is stale.